File: src/main/java/com/hank/lottery/BigGame.java
package com.hank.lottery;

public class BigGame extends NumberGame{
    int numberCount = 6;

    @Override
    public void generate() {
        super.numberCount = numberCount;
        numbers.add(99);
        numbers.add(88);
        numbers.add(77);
        numbers.add(66);
        numbers.add(55);
        numbers.add(44);
        System.out.println("大樂透 : set長度： " + numbers.size());
        System.out.println("需求長度是 " + numberCount +"：" + validate());
    }
}

File: src/main/kotlin/com/hank/lottery/BigGame.kt
package com.hank.kotlin

class BigGame:NumberGame() {

    override fun generate() {
        val numberCountBig = 6
        super.numberCount = numberCountBig
        numbers.add(101)
        numbers.add(201)
        numbers.add(301)
        numbers.add(401)
        numbers.add(501)
        println("kotlin BigGame set長度=${numbers.size}")
        println("kotlin BigGame 需求長度=$numberCountBig")
        println("kotlin BigGame 選號數量是否正常=${validate()}")
    }
}